File: app/src/main/java/com/babestudios/companieshouse/data/model/company/AccountingRequirement.java
package com.babestudios.companieshouse.data.model.company;


import com.google.gson.annotations.SerializedName;

public class AccountingRequirement {

	@SerializedName("foreign_account_type")
	public String foreignAccountType;

	@SerializedName("terms_of_account_publication")
	public String termsOfAccountPublication;
}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/company/AccountingRequirement.kt
package com.babestudios.companyinfouk.data.model.company


import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
class AccountingRequirement(

		@SerializedName("foreign_account_type")
		var foreignAccountType: String? = null,

		@SerializedName("terms_of_account_publication")
		var termsOfAccountPublication: String? = null
) : Parcelable
